src/mini_trainer/train.py

```diff
@@ -343,7 +359,15 @@ def train(
 
                 # torch.distributed.breakpoint()
                 output = model(**model_inputs)
-                loss = output.loss.float().sum() 
+                
+                # GPT-OSS: add auxiliary loss if present, otherwise use standard loss
+                if hasattr(output, 'aux_loss') and output.aux_loss is not None:
+                    # GPT-OSS model: add auxiliary loss to main loss
+                    loss = output.loss.float().sum() + output.aux_loss.float()
+                else:
+                    # Standard model: use existing loss computation
+                    loss = output.loss.float().sum()
```

Comment:
Nit: since the regular loss is summed up in either case, it's good to eliminate the amount of conditional branching to simplify control flow:


```suggestion
                # GPT-OSS: add auxiliary loss if present, otherwise use standard loss
                loss = output.loss.float().sum()
                if hasattr(output, 'aux_loss') and output.aux_loss is not None:
                    # GPT-OSS model: add auxiliary loss to main loss
                    loss += output.aux_loss.float()
```